modules/tfm/src/zephyr_tfm_psa_test.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+
+/**
+ * \brief This symbol is the entry point provided by the PSA API compliance
+ *        test libraries
+ */
+extern void val_entry(void);
+
+/**
+ * \brief This symbol is the TIMER1 IRQ handler used by the TFM PSA test.
+ */
+extern void TIMER1_Handler(void);
+
+__attribute__((noreturn))
+void main(void)
+{
+	IRQ_CONNECT(TIMER1_IRQn, NRFX_TIMER_DEFAULT_CONFIG_IRQ_PRIORITY,
+		TIMER1_Handler, NULL, 0);
```

Comment:
What is this line for?

Reply:
It's nRF-specific. Added an ifdef now.

Reply:
There's a timer set up in the test code that needs this.

Reply:
Questions:
1) Since there is a timer set up in code, is not possible use this ?https://docs.zephyrproject.org/latest/reference/kernel/timing/timers.html
2) What differentiate nRF from other platforms?

Reply:
The timer is in trusted-firmware-m so it cannot depend on Zephyr.

I haven't been able to try all platforms, since I don't have hardware, but the AN521 on QEMU works fine without.

I could remove it in this PR, though, since nRF isn't strictly speaking supported until the next upmerge of trusted-firmware-m.

Reply:
Removed